pootle_pytest/fixtures/site.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+import shutil
+
+import pytest
+
+
+@pytest.fixture
+def site_matrix(request, system, settings):
+
+    from pootle_pytest.factories import (
+        ProjectFactory, DirectoryFactory, LanguageFactory,
+        TranslationProjectFactory, StoreFactory, UnitFactory)
+
+    from pootle_store.models import UNTRANSLATED, TRANSLATED, FUZZY, OBSOLETE
```

Comment:
@unho im not commenting this or else i would need to comment it in a 100 places

if you write tests - you know